app/javascript/components/CreateWork.jsx

```diff
@@ -64,13 +110,21 @@ class CreateWork extends React.Component {
     var currentImages = this.state.work.images;
     const [newImages] = images;
     currentImages.push(newImages);
+    console.log(images)
 
     // Assign files dropped into component into state
     var work = this.state.work;
     work.images = currentImages;
     this.setState({ work: work });
   }
 
+  selectFeaturedImage(img) {
+    var work = this.state.work;
+    work.featured_image = img;
+    this.setState({work: work});
+    console.log(work);
```

Comment:
Can you remove all the puts and the logs?

Reply:
fixed!